### PR TITLE
Miscellaneous Improvements

### PR DESCRIPTION
**Pull request by ianardee**
# Miscellaneous Improvements

Each commit should be kept, do not squash this branch.

## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.

### CODE REVIEW

**Review by fharper — CHANGES_REQUESTED** [review]
So I have two unanswered questions in the code + the tests don't work, probably because of the latest 2 commits

File: mindee/documents/passport.py [hunk 1]
```diff
@@ -133,23 +133,9 @@ def _checklist(self) -> None:
             "mrz_expiration_date_checksum": self.__mrz_expiration_date_checksum(),
             "mrz_personal_number_checksum": self.__mrz_personal_number_checksum(),
             "mrz_last_name_checksum": self.__mrz_last_name_checksum(),
-            "mrz_checksum": self.__mrz__checksum(),
         }
 
     # Checks
-    def __mrz__checksum(self) -> bool:
```
[the remark this answers is not in the document]
Comment (ianardee) [reply mindee/documents/passport.py:140]: it's dead code

[the remark this answers is not in the document]
Comment (fharper) [reply mindee/documents/passport.py:140]: But we don't do the checksum anymore or maybe it's done at the API level?

[the remark this answers is not in the document]
Comment (ianardee) [reply mindee/documents/passport.py:140]: check is done just above
https://github.com/mindee/mindee-api-python/pull/82/commits/9d30163b23ad61549d3186b0ec74ec80c9b305dc#diff-b172e3b73e7c1f3e15da36dfc1a0f83d8ca196db99d2e1fa52a3f967639c51f9R130

**Pull request merged**